### DISCUSSION TITLE
2011/07/16/%E3%80%8A%E5%AF%BB%E8%B7%AF%E4%B8%AD%E5%9B%BD%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#339] [Announcements]
# 2011/07/16/%E3%80%8A%E5%AF%BB%E8%B7%AF%E4%B8%AD%E5%9B%BD%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

有一阵子没怎么读书了。最近找到了一个比较适合阅读的零散时间：上下班的地铁上。为了方便携带，我还特意买了一部Kindle（电纸书）。Kindle到手后，就随意在网上找了几部书来看。可能这本《寻路中国》是比较热门的新书，最先下载的几本书里就有它。

https://blog.qizhen.xyz/2011/07/16/%E3%80%8A%E5%AF%BB%E8%B7%AF%E4%B8%AD%E5%9B%BD%E3%80%8B%E8%AF%BB%E5%90%8E%E6%84%9F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

谈天
2011年08月1日 09:23 编辑
你现在不开车上下班？

回复
ruanqizhen
2011年08月1日 09:35 编辑
为了方便给豆豆喂奶，我在老婆单位的隔壁租了个房间。从那里上班还是地铁方便。

回复
adeline
2011年07月18日 11:17 编辑
这本书我在Kindle上买的，我觉得挺值的。国人的习性，能透过这些外国人了解得更深刻的话，也是好事啊。Peter的夫人Leslie T. Chang张彤禾，写了一本Factory Girls，也很好看。

回复
ruanqizhen
2011年07月18日 13:00 编辑
哇，你连他家室都研究清楚啦。我看他那本书有个没搞清楚的，就是那个Mimi跟他是啥关系呢？

回复
ronin
2011年07月17日 23:04 编辑
上次京东大降价想买一本的，网上没有电子版

回复
